src/support_sphere_py/tests/resources/scripts/update_db_sample_data.py

```diff
@@ -72,11 +72,12 @@ def populate_cluster_and_household_details():
 
 def get_supabase_client() -> Client:
 
+    logger.info("Establishing connection via supabase")
     # Setting up the supabase client for python
     file_path = Path("./deployment/values.dev.yaml")
     with file_path.open(mode='r') as file:
         config = yaml.safe_load(file)
-        url = config['studio']['environment']['SUPABASE_PUBLIC_URL']
+        url = 'http://supabase-supabase-kong:8000/'
```

Comment:
This should still take the regular supabase public url.
```suggestion
        url = os.environ.get("SUPABASE_PUBLIC_URL")
```